sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -25,6 +25,12 @@
     'asia-east1',
 ]
 
+# Default instance family for CPU-only VMs.
```

Comment:
Add: This is the latest general-purpose instance family as of Jan 2023.

Reply:
Added.